.github/workflows/test.yml

```diff
@@ -112,4 +112,7 @@ jobs:
                   make -C kind setup
                   make -C kfp/kfp_support_lib test
                   make -C transforms workflow-build
-                  make -C transforms/universal/noop workflow-test
+                  dir=("code"  "universal") && index=$(($RANDOM % 2)) && subdirs=${dir[$index]} && transforms=($(find transforms/$subdirs/ -type d  -maxdepth 1 ))
+                  # First element is not really a subdir but rather the current dir so remove it and randomly choose a transform to run
+                  set -- "${transforms[@]}" && shift && transforms=("$@") && size=${#transforms[@]} && index=$(($RANDOM % $size))
+                  make -C ${transforms[$index]} workflow-test
```

Comment:
does it print the chosen workflow name?

Reply:
Yes. please see the following examples: https://github.com/revit13/data-prep-lab/actions/workflows/debug_workflow.yml

Reply:
I don't see it at https://github.com/revit13/data-prep-lab/actions/runs/9535702057/job/26281751327, after all `kubectl describe pods` ...

Reply:
Sorry i put the link to the manual run by mistake... please see the end of: `Test CI / test-kfp-workflow-run (pull_request) Successful in 20m` run in this PR

```
12:45:14 INFO - Pipeline uploaded
12:45:14 INFO - Pipeline run filter_wf 2024_06_16_12_45_14 submitted
12:46:14 INFO - Got pipeline execution status Running
12:47:14 INFO - Got pipeline execution status Succeeded
12:47:14 INFO - Pipeline filter_wf successfully completed
```